image-layout.md

```diff
@@ -0,0 +1,47 @@
+## Open Container Initiative Image Layout Specification
+
+The OCI Image Layout is a slash separated layout of OCI content-addressable objects and human significant references (refs) 
+This layout MAY be used in a variety of different transport mechanisms from archive formats (e.g. tar, zip) or used in shared filesystem environments (e.g. nfs) or in networked file fetching systems (e.g. http, ftp, rsync).
+Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appopriate manifest from the manifest list, unpacking the filesystem serializations in the correct order, and then converting the image configuration into an OCI Runtime config.json.
+
+The image layout has two top level directories: 
+
+- "blobs" contains content-addressable blobs
```

Comment:
On Fri, May 27, 2016 at 12:35:56PM -0700, Stephen Day wrote:
> > +- "blobs" contains content-addressable blobs
>
> I am defending the use of the term "blobs" and wondering if we
> should enshrine said defense…

How about:

  The image layout does not specify an internal structure for blobs;
  any semantics must be inferred from the blob content or the context
  from which the blob was referenced.